### PR TITLE
QGA: increase fsfreeze session.cmd timeout

### PR DESCRIPTION
**Pull request by BohdanMar**
Qemu_Guest_Agent: increase fsfreeze session.cmd timeout

ID: 1949

Signed-off-by: bmarcynk bmarcynk@redhat.com

### CODE REVIEW

**Comment by BohdanMar** [comment]
Tested on aarch64 platform:

JOB ID     : 16a31488075cc93360f6a88e126043beaabadf8a
JOB LOG    : /root/avocado/job-results/job-2024-02-08T09.02-16a3148/job.log Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_64k.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsfreeze.arm64-pci: PASS

File: qemu/tests/qemu_guest_agent.py [hunk 1]
```diff
@@ -2312,7 +2312,7 @@ def _guest_cmd_run(guest_cmd, cmd_args=None, env_qga=None,
     def _action_before_fsfreeze(self, *args):
         session = self._get_session(self.params, None)
         if self.params.get("os_type") == "linux":
-            session.cmd("restorecon -Rv /", timeout=180)
+            session.cmd("restorecon -Rv /", timeout=360)
```
Comment (PaulYuuu) [inline qemu/tests/qemu_guest_agent.py:2315]: The original problem is only under `/mnt`, so I think here we can limit to restorecon in the `/mnt` dir to reduce time. Can you try?

Comment (BohdanMar) [reply qemu/tests/qemu_guest_agent.py]: Hi @PaulYuuu 
Yes, it makes sense, thanks for the review. 
Updated.

JOB ID     : b3dc2ead62b9c9dc7675d7a82a11d59bf9f9965a
JOB LOG    : /root/avocado/job-results/job-2024-02-09T09.29-b3dc2ea/job.log
 Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_64k.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsfreeze.arm64-pci: STARTED
 Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_64k.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsfreeze.arm64-pci: PASS

File: qemu/tests/qemu_guest_agent.py [hunk 2]
```diff
@@ -2312,7 +2312,7 @@ def _guest_cmd_run(guest_cmd, cmd_args=None, env_qga=None,
     def _action_before_fsfreeze(self, *args):
         session = self._get_session(self.params, None)
         if self.params.get("os_type") == "linux":
-            session.cmd("restorecon -Rv /", timeout=180)
+            session.cmd("restorecon -Rv /mnt", timeout=360)
```
Comment (PaulYuuu) [inline qemu/tests/qemu_guest_agent.py:2315]: Should we don't need to change the timeout here.

Comment (BohdanMar) [reply qemu/tests/qemu_guest_agent.py]: Updated:

JOB ID     : 6f41b60a7cd9f6f2543ddc6480038f323d158700
JOB LOG    : /root/avocado/job-results/job-2024-02-27T09.02-6f41b60/job.log
Host_RHEL.m9.u4.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.aarch64.page_64k.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_fsfreeze.arm64-pci: PASS

**Comment by BohdanMar** [comment]
@PaulYuuu Thanks for pointing this out. Done

**Comment by BohdanMar** [comment]
I will close this pull request since it duplicates: https://github.com/autotest/tp-qemu/pull/3980

**Pull request closed without merging**